roles/openshift_certificate_expiry/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Check cert expirys on host
+  openshift_cert_expiry:
+    warning_days: "{{ openshift_certificate_expiry_warning_days|int }}"
+    config_base: "{{ openshift_certificate_expiry_config_base }}"
+    show_all: "{{ openshift_certificate_expiry_show_all|bool }}"
+  register: check_results
+
+- name: Generate expiration report HTML
+  become: no
+  run_once: yes
+  template:
+    src: cert-expiry-table.html.j2
+    dest: "{{ openshift_certificate_expiry_html_report_path }}"
+  delegate_to: localhost
+  when: "{{ openshift_certificate_expiry_generate_html_report|bool }}"
+
+- name: Generate expiration results JSON
+  become: no
+  run_once: yes
+  template:
+    src: save_json_results.j2
```

Comment:
@tbielawa main thing I was thinking was the ability to modify as a python dict and serialize directly to json for output to avoid maintenance issues down the line rather than relying on the template to always produce valid json.